content/en/docs/reference/kube-scheduler/profiles.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Kube-scheduler Profiles
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
+
+A scheduling Profile allows to configure the different stages of scheduling
+in [kube-scheduler](/docs/reference/command-line-tools-reference/kube-scheduler/).
+Each stage is exposed in a extension point. Plugins provide scheduling behaviors
+by implementing one or more of these extension points.
+
+You can specify scheduling profiles for kube-scheduler in a configuration file
+(via `--config` flag) using the component config APIs
+([`v1alpha1`](https://pkg.go.dev/k8s.io/kube-scheduler@v0.18.0/config/v1alpha1?tab=doc#KubeSchedulerConfiguration)
+or [`v1alpha2`](https://pkg.go.dev/k8s.io/kube-scheduler@v0.18.0/config/v1alpha2?tab=doc#KubeSchedulerConfiguration)).
```

Comment:
@alculquicondor , Double check this link (will the package link resolve once the release is finalized?), 
https://pkg.go.dev/k8s.io/kube-scheduler/config/v1alpha1?tab=versions

Reply:
Yes, it should. But we can't really control that, I would say.
Because of that, I opened kubernetes/kubernetes#88919